scripts/.env.template

```diff
@@ -1,3 +1,4 @@
 FTP_HOST = 
 FTP_USERNAME = 
 FTP_PASSWORD =
+GCP_VISION_API_KEY =
```

Comment:
@ilyasubkhankulov should the spaces around the equals signs be there? I followed your lead, but I always thought that .env files should *not* have spaces.

Reply:
I think thats right but for this particular library, it looks like spaces are ok:
https://pypi.org/project/python-dotenv/#:~:text=Spaces%20before%20and%20after%20keys%2C%20equal%20signs%2C%20and%20values%20are%20ignored.